### ISSUE TITLE
[Bug]: [a11y] "Set as default browser" button is read as "Button. Double-tap to activate" on Android 5 device.

### ISSUE THREAD

**Issue by delia-pop** [mozilla-mobile/fenix#25188]
### Steps to reproduce

Preconditions:
The default browser messaging banner is displayed on homepage.
TalkBack (or other screen reader application) is installed and enabled.

1. From homepage, tap on the "Set as default browser button".
2. Listen what the screen reader announces.

### Expected behaviour

The TalkBack announces: "Set as default browser button. Double tap to activate".

### Actual behaviour

The TalkBack announces: "Button. Double tap to activate". The text "Set as default browser button" is not announced.

### Device name

Huawei MediaPad M2

### Android version

Android 5.1.1.

### Firefox release type

Firefox Beta

### Firefox version

Beta 101.0.0-beta.3 

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FNXV2-20430)
